cylc/flow/suite_files.py

```diff
@@ -611,6 +656,77 @@ def clean(reg):
     _remove_empty_reg_parents(reg, run_dir)
 
 
+def remote_clean(reg, platform_names):
+    """Run subprocesses to clean workflows on remote install targets
+    (skip localhost), given a set of platform names to look up.
+
+    Args:
+        reg (str): Workflow name.
+        platform_names (list): List of platform names to look up in the global
+            config, in order to determine the install targets to clean on.
+    """
+    try:
+        install_targets_map = (
+            get_install_target_to_platforms_map(platform_names))
+    except PlatformLookupError as exc:
+        raise PlatformLookupError(
+            "Cannot clean on remote platforms as the workflow database is "
+            f"out of date/inconsistent with the global config - {exc}")
+
+    queue = []
+    for target, platforms in install_targets_map.items():
+        if target == 'localhost':
+            continue
+        shuffle(platforms)
+        # Issue ssh command:
+        queue.append(
+            (_remote_clean_cmd(reg, platforms[0]), target, platforms)
+        )
```

Comment:
`random.choice(_list_)` might create cleaner code?
```suggestion
        # Issue ssh command:
        queue.append(
            (_remote_clean_cmd(reg, choice(platforms), target, platforms)
        )
```

Reply:
Would be better for load balancing but would have to be done earlier in the code as in the case of failure the platform has to be popped from the list.

Reply:
I'm not sure what you mean by it would be better for load balancing?